services/api-server/src/simcore_service_api_server/api/routes/users.py

```diff
@@ -22,11 +23,12 @@ async def get_my_profile(
     data = await client.get("/me")
 
     # FIXME: temporary patch until web-API is reviewed
-    data["role"] = data["role"].upper()
+
     try:
+        data["role"] = data["role"].upper()  # type: ignore
         profile = Profile.parse_obj(data)
-    except ValidationError as err:
-        logger.exception("webserver invalid response")
+    except (ValidationError, KeyError) as err:
+        logger.error("webserver invalid response %s", data, exc_info=True)
```

Comment:
Maybe rethink theology message a well. What does `webserver invalid response` mean to someone looking in the logs?
The reason seems to be related to invalid request data received.

Reply:
Also the role.upper() could be handled by a validator in the Profile model at this point.

Reply:
@GitHK  I agree. The whole error handling in the api-server is very very poor :-(  
I focused here in mypy but will review all error handling and logging in the api-server